routes/index.js

```diff
@@ -1,9 +1,13 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
 
 /* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+router.get("/", function (req, res, next) {
+  res.render("index", { title: "Dairy Farm Account" });
+});
+
+router.get("/login", function (req, res, next) {
+  res.render("index", { title: "Dairy Farm Account" });
```

Comment:
This should not render index, it should render a login template and not index template